project_management/turing_mybinder_role.md

```diff
@@ -0,0 +1,51 @@
+# Turing mybinder.org Cluster Admin Role
+
+This document outlines the minimal responsibilities and requirements for the technical administrator of the BinderHub deployment _The Turing Way_ donates to the [Binder Federation](https://blog.jupyter.org/the-international-binder-federation-4f6235c1537e).
+
+---
+
+## Context
+
+_The Turing Way_ supports and maintains a Kubernetes cluster that is part of the Binder Federation, facilitating the decentralisation of the mybinder.org service.
+This cluster also hosts Hub23, a private BinderHub for the Turing research community.
+Maintaining this infrastructure requires technical skill and a small time commitment that are detailed below.
+
+It is important to the Binder Operating team that we are not anonymous GitHub handles to one another.
+Knowing who we are and why we're here helps us support each other and connect those who can help each other.
+Please do say hi and [introduce yourself](https://discourse.jupyter.org/t/introduce-yourself/17)! :smile:
+
+## Requirements
+
+- An account in the Turing's Azure tenancy
```

Comment:
Might be worth adding who can apply for these / how people apply for these (as you've done with the point below)

Reply:
Addressed in f591545

Reply:
> Might be worth adding who can apply for these / how people apply for these (as you've done with the point below)

There are a couple of internal people I'd like to approach first - but if neither of them are interested, I'll add a more formal "how to apply" section with guidance from @malvikasharan